integration_tests/test_page_loads.py

```diff
@@ -32,7 +31,7 @@
 
 
 @pytest.fixture(scope="module", autouse=True)
-def auto_populate_index(populated_index: Index):
+def auto_populate_index(populated_index):
```

Comment:
should this be called `_auto_populate_index` for consistent naming conventions?

Reply:
Fixed by deleting :D